src/userscores/qml/MuseScore/UserScores/ExportDialog.qml

```diff
@@ -0,0 +1,237 @@
+                            Layout.fillWidth: true
+                            Layout.preferredHeight: childrenRect.height
+
+                            spacing: 6
+
+                            StyledTextLabel {
+                                Layout.fillWidth: true
+                                Layout.topMargin: 24
+
+                                text: qsTrc("userscores", "Export As: ")
+                                font.capitalization: Font.AllUppercase
+
+                            }
+
+                            StyledComboBox {
+                                id: fileTypeSelect
+
+                                property var currValue: 1
+
+                                width: rightRectangle.width * 0.8
+
+                                textRoleName: "text"
+                                valueRoleName: "value"
+
+                                model: [
+                                    { text: "PDF File", value: 1, suffix: "pdf" },
+                                    { text: "PNG Image", value: 2, suffix: "png" },
+                                    { text: "SVG Image", value: 3, suffix: "svg" },
+                                    { text: "MIDI", value: 4, suffix: "midi" },
+                                    { text: "mp3", value: 5, suffix: "mp3" },
+                                    { text: "ogg", value: 6, suffix: "ogg" },
+                                ]
+
+                                currentIndex: indexOfValue(currValue)
+
+                                onValueChanged: {
+                                    currValue = value;
+                                    scoresModel.setExportSuffix(valueFromModel(indexOfValue(value), "suffix"));
+                                    fileExportPathInput.currentText = scoresModel.getExportPath();
+                                }
+                            }
+                        }
+
+
+                        ExportFilePages {
+                            id: exportFileStackPages
+                            model: scoresModel
+                        }
+
+                    }
+
+                }
+
+            }
+
+            Row {
+                Layout.preferredHeight: childrenRect.height
+                Layout.bottomMargin: privateProperties.buttonsMargin
+                Layout.rightMargin: privateProperties.buttonsMargin
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
+
+                spacing: 12
+
+                FlatButton {
+                    text: qsTrc("global", "Close")
+
+                    onClicked: {
+                        root.hide()
+                    }
+                }
+
+                FlatButton {
+                    id: exportButton
+
+                    text: qsTrc("global", "Export")
+
+                    normalStateColor: ui.theme.accentColor
+                    hoveredStateColor: ui.theme.accentColor
+                    pressedStateColor: ui.theme.accentColor
```

Comment:
You can use accentButton property in such situations

Before:
normalStateColor: ui.theme.accentColor
hoveredStateColor: ui.theme.accentColor
pressedStateColor: ui.theme.accentColor

After:
accentButton: true